### PR TITLE
Tangobox documentation update

### PR DESCRIPTION
**Pull request by piogor**
This updates virtual machine related docs for TangoBox 9.3.3 release.

### CODE REVIEW

**Comment by piogor** [comment]
The screenshots still needs to be updated...

File: source/installation/vm/tangobox.rst [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+on for being explored, see :ref:`Switching containers on and off <container_switch_on_off>`
+
+* Tango 9.3.3
+* Tango Access Control
+* PyTango 9.3.3
+* Taurus 4.5.1
+* QTango
+* Cumbia
+* ITango
+* Sardana 2.5.0
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: 2.7.0

File: source/installation/vm/tangobox.rst [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+* Docker
+* :ref:`Linac system simulation <jlinac>` (as docker container tangobox-sim)
+* :ref:`HDB/TDB, SNAP DS <hdb_tdb_snap>` (as docker container tangobox-archiving)
+* :ref:`HDB++ <hdbpp>` (as docker container tangobox-hdbpp)
+* :ref:`JupyTango <jupytango>` (as docker container tangobox-jupytango)
+* SerialLine, Modbus and PyPLC device server (as docker container tangobox-com)
+* :ref:`mTango + restAPI <rest_api>`, :ref:`Tango WebApp <webapp>` (as docker container tangobox-web)
+* :ref:`e-giga` (as docker container tangobox-egiga)
+* PyCharm
+* Visual Studio Code
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: Not present

File: source/installation/vm/tangobox.rst [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+* :ref:`ModbusPal to simulate Modbus <Modbus>`
+
+First steps
+-----------
+
+* First of all you have to download latest release of VirtualBox. It can be downloaded from `www.virtualbox.org <https://www.virtualbox.org/>`_ .
+  Simply install it and start the program.
+* TangoBox is released in **.ova** extension so it can be easily imported.
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: Consider: ".ova package"

File: source/installation/vm/tangobox.rst [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+      You may change number of CPUs and increase RAM size. Memory size has major impact on VM performance.
+
+* Wait for VirtualBox to import machine
+
+After importing the VM image to VirtualBox you may start it.
+
+* Username is: `tango-cs`
+* Password is: `tango`
+
+`tango-cs` user has sudo rights, so he may invoke commands as superuser with command :command:`sudo`.
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: Consider: "has sudo privileges"

File: source/installation/vm/tangobox.rst [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+Each container is based on its image. All images are already build but, if neccessary, *Dockerfiles* are stored in :file:`home/Dockerfiles`
+directory. Below is the list of all containers and corresponding images:
+
+================== =========== ===============================
+Container            Image             Remarks
+================== =========== ===============================
+tangobox-com         com                 -
+tangobox-sim         sim                 -
+tangobox-archiving   archive              -
+tangobox-hdbpp       hdbpp               -
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: The images are hosted in a registry, so above image name is no longer valid. See:

```
CONTAINER ID        IMAGE                                                                        COMMAND                  CREATED             STATUS              PORTS               NAMES
04e048c14dba        registry.gitlab.com/s2innovation/tangobox-docker/tangobox-hdbpp:latest       "/bin/sh -c '/usr/bi…"   12 days ago         Up 8 minutes                            tangobox-hdbpp
```

File: source/installation/vm/tangobox.rst [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+JupyTango is a `Jupyter <http://jupyter.org/>`_ featuring Tango related kernels. With JupyterLab you may interact
+and do scripting for Tango through a web browser.
+
+.. figure:: tangobox/jupytango.png
+
+   Browser window with JupyTango in action
+
+**In case you want to try it, here's the procedure:**
+
+1. start jupyterlab using our dedicated script: `jupytango`
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: It's now `docker start tangobox-jupytango` (it is started by default)

File: source/installation/vm/tangobox.rst [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+2. a new browser tab is automagically opened with the right URL: :file:`localhost:8888/lab?`. Please be patient, it may
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: It's `tangobox-jupytango:8888/lab` now

File: source/installation/vm/tangobox.rst [hunk 8]
```diff
@@ -0,0 +1,367 @@
...
+* `-w` or `--width`: plot width in pixels
+* `-h` or `--height`: plot height in pixels
+* `-p` or `--period`: plot refresh period in [0.1, 5] seconds - defaults to 1s
+* `-d` or `--depth`: scalar attribute history depth in [1, 3600] seconds - defaults to 900s
+
+You can try to kill the monitored device will the JupyTango monitor is running to see how errors are handled.
+
+.. _jlinac:
+
+JLinac and Elinac simulation
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: Elinac is no longer included

File: source/installation/vm/tangobox.rst [hunk 9]
```diff
@@ -0,0 +1,367 @@
...
+* `tangobox-archive` and `tangobox-web` containers must be started and archiving device servers must be running
+* use i.e. **Mambo** to enable data archiving for HDB database. It is required. If you do not see any attributes in
+  `E-giga` it is probably due to archiving being disabled. Check with :program:`Astor` if the tangobox-archive LED
+  is green and with :program:`Mambo` if there are any attributes configured to be archivied.
+
+To open browser with :program:`E-giga` click on the relate desktop icon.
+
+.. note::
+
+   Please keep in mind that you should not rebuild **tangobox-web** image because its configuration is not included in Dockerfile
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: I think it is not true anymore

File: source/installation/vm/tangobox.rst [hunk 10]
```diff
@@ -0,0 +1,367 @@
...
+
+.. _rest_api:
+
+REST API
+~~~~~~~~
+
+Tango Controls specifies REST API interface and provides its reference implementation. For details see
+:ref:`REST API documentation <tango_rest_api>`
+
+The **TangoBox** comes with REST API installed. The `tangobox-web` container must be started to play with it. Invoke
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: Rest server has been moved to the host machine. It is no longer included in tangobox-web container.

File: source/installation/vm/tangobox.rst [hunk 11]
```diff
@@ -0,0 +1,367 @@
...
+
+Sardana is a software suite for Supervision, Control and Data Acquisition in scientific installations. I
+t aims to reduce cost and time of design, development and support of the control and data acquisition systems.
+For more information about it please refer to `Sardana documentation <http://www.sardana-controls.org>`_.
+
+.. figure:: tangobox-9.2/sardana.png
+
+   SardanaGUI in action
+
+To play with Sardana the **tangobox-sardana** container has to be started. Open a terminal and call
```
Comment (mliszcz) [inline source/installation/vm/tangobox.rst]: It runs on host now (as there is a common package for client & server)

**Pull request merged**